index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Josh Lee</title>
+
+  <!-- Leave these links here...
+  But this should be in a comment... -->
+  <link href="https://fonts.googleapis.com/css?family=Cormorant|Poppins:300,400" rel="stylesheet"/>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link type="text/css" rel="stylesheet" href="style.css"/>
```

Comment:
Check the path of `style.css` try putting renaming the file to `styles/style.css` and updating this link to match the path names in Newline